tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/Indexing.java

```diff
@@ -0,0 +1,206 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.tensorflow.op.core;
+
+import org.tensorflow.Operand;
+import org.tensorflow.op.Index;
+import org.tensorflow.op.Index.Singular;
+import org.tensorflow.op.Scope;
+import org.tensorflow.op.annotation.Endpoint;
+import org.tensorflow.op.annotation.Operator;
+import org.tensorflow.types.family.TType;
+
+/**
+ * Helper endpoint methods for Python like indexing.
+ *
+ * @see Index
+ */
+@Operator
+public class Indexing {
+
+  static class StridedSliceArgs {
+
+    final int[] begin;
+    final int[] end;
+    final int[] strides;
+    final long beginMask;
+    final long endMask;
+    final long ellipsisMask;
+    final long newAxisMask;
+    final long shrinkAxisMask;
+
+    public StridedSliceArgs(int[] begin, int[] end, int[] strides, long beginMask, long endMask, long ellipsisMask,
```

Comment:
Does this need to be public? Can't it be package private?

Reply:
Yeah, it should be.